driver-legacy/src/main/com/mongodb/DB.java

```diff
@@ -693,7 +696,9 @@ public void slaveOk() {
      *
      * @param option value to be added
      * @mongodb.driver.manual ../meta-driver/latest/legacy/mongodb-wire-protocol/#op-query Query Flags
+     * @deprecated Replaced with {@link DBCursor#addOption(int)}
      */
+    @Deprecated
```

Comment:
I don't feel that strongly about these options-related methods.  I just was reminded of them because they use Bytes.OptionsHolder, which will be removed, and I suspect they are rarely used.

Reply:
Agreed - we should remove them